packages/payment-processor/package.json

```diff
@@ -45,6 +45,7 @@
     "@requestnetwork/smart-contracts": "0.29.0",
     "@requestnetwork/types": "0.36.0",
     "@requestnetwork/utils": "0.36.0",
+    "@requestnetwork/multi-format": "0.15.10",
```

Comment:
Per [lint error](https://app.circleci.com/pipelines/github/RequestNetwork/requestNetwork/7804/workflows/246b385c-1ec2-4a4f-9396-98152fe9a913/jobs/127334?invite=true#step-102-7): Please re-arrange dependencies to be in alphabetical order.